### PR TITLE
[docs] [integrations] removed YugabyteDB JDBC driver from integrations

### LINKED ISSUE #13388: [Docs] [Integrations] Remove duplicate pages from integrations
### Description

Some pages in Integrations are now under Drivers and ORMs/ and References. Check and removes the ones that duplicates.

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #13388
YugabyteDB JDBC driver was recently moved under this section: https://docs.yugabyte.com/preview/reference/drivers/java/yugabyte-jdbc-reference/
It's no longer required under integrations.

### CODE REVIEW

File: docs/netlify.toml [hunk 1]
```diff
@@ -261,12 +261,12 @@
...
 [[redirects]]
   from = "/:version/integrations/smart-driver/"
-  to = "/preview/integrations/jdbc-driver/"
+  to = "/preview/reference/drivers/java/yugabyte-jdbc-reference/"
   force = true
 
 [[redirects]]
-  from = "/:version/integrations/*"
-  to = "/preview/integrations/:splat"
+  from = "/:version/integrations/*-driver/"
```
Comment (polarweasel) [inline docs/netlify.toml:268]: This one should happily catch what the previous is doing. Try commenting out the redirect that starts on line 262 and make sure everything still works as intended.

Comment (polarweasel) [reply docs/netlify.toml:268]: (And try a URL that matches `/:version/integrations/*` just to make sure the `-driver` is actually doing something! Sometimes regex implementations suck...)

**Review by ddhodge — APPROVED** [review]
There are 5 or 6 links to integrations/jdbc-driver in the docs, maybe update those to point to drivers and orms or reference jdbc driver page, depending on the context

File: docs/content/preview/develop/build-apps/java/ysql-yb-jdbc.md [hunk 2]
```diff
@@ -365,4 +365,4 @@ When using `DriverManager.getConnection()`, you need to include the `load-balanc
 
 ## Explore the driver
 
-Learn more about the [Yugabyte JDBC driver](../../../../integrations/jdbc-driver/) and explore the [demo apps](https://github.com/yugabyte/pgjdbc/tree/master/examples) to understand the driver's features in depth.
+Learn more about the [YugabyteDB JDBC driver](../../../../integrations/jdbc-driver/) and explore the [demo apps](https://github.com/yugabyte/pgjdbc/tree/master/examples) to understand the driver's features in depth.
```
Comment (ddhodge) [inline docs/content/preview/develop/build-apps/java/ysql-yb-jdbc.md:368]: update link

File: docs/content/preview/integrations/camunda.md [hunk 3]
```diff
@@ -58,19 +56,15 @@ This tutorial assumes that:
...
+    Then, download the YugabyteDB JDBC driver [JAR file](https://repo1.maven.org/maven2/com/yugabyte/jdbc-yugabytedb/42.3.5-yb-1/jdbc-yugabytedb-42.3.5-yb-1.jar) and place it in the `camunda-bpm-run-7.17.0/configuration/userlib` directory. Read more about the YugabyteDB JDBC driver on its [documentation page](../../reference/drivers/java/yugabyte-jdbc-reference/).
 
 1. Start the Camunda Platform server using `./start.sh` on Linux or macOS, or `./start.bat` on Windows.
 
 ## Verify the integration
 
 After the server is started, you'll be able to see a few tables getting created in the database. To verify, do the following:
 
-1. Login to the database using `./bin/ysqsh`
+1. Login to the database using `./bin/ysqsh`.
```
Comment (ddhodge) [inline docs/content/preview/integrations/camunda.md:67]: Log in

File: docs/content/preview/integrations/camunda.md [hunk 4]
```diff
@@ -194,14 +170,12 @@ To deploy the process, do the following:
 
 1. Click the deploy button in Camunda Modeler, enter the deployment name "Payment Retrieval", and click the Deploy button.
 
-    \
-    If you're running Camunda Modeler version 3.0.0 or later, you need to provide the URL of an Endpoint Configuration along with the deployment details. This can be either the root endpoint to the REST API (such as <http://localhost:8080/engine-rest>) or an exact endpoint to the deployment creation method (such as <http://localhost:8080/engine-rest/deployment/create>). <br/><br/>
+    If you're running Camunda Modeler version 3.0.0 or later, you need to provide the URL of an Endpoint Configuration along with the deployment details. This can be either the root endpoint to the REST API (such as <http://localhost:8080/engine-rest>) or an exact endpoint to the deployment creation method (such as <http://localhost:8080/engine-rest/deployment/create>).
 
     ![alt text](https://docs.camunda.org/get-started/quick-start/img/modeler-deploy2.png)
```
Comment (ddhodge) [inline docs/content/preview/integrations/camunda.md:175]: image doesn't display - need to add camunda.org domain

File: docs/content/preview/reference/configuration/yb-master.md [hunk 5]
```diff
@@ -318,7 +318,7 @@ For information on YB-Master load balancing, see [Data placement and load balanc
 
 For load balancing commands in `yb-admin`, see [Rebalancing commands (yb-admin)](../../../admin/yb-admin/#rebalancing-commands).
 
-For information on internal load balancing to power geo-distributed applications, see [Yugabyte JDBC Driver](../../../integrations/jdbc-driver).
+For information on internal load balancing to power geo-distributed applications, see [YugabyteDB JDBC Driver](../../../reference/drivers/java/yugabyte-jdbc-reference/).
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-master.md:321]: Correct link? Section is about internal data load balancing pretty sure; placement of tablets/shards across the nodes, ie not connection load balancing. Not sure it's all that relevant to drivers, or at least only peripherally

File: docs/content/v2.12/reference/configuration/yb-master.md [hunk 6]
```diff
@@ -340,7 +340,7 @@ For information on YB-Master load balancing, see [Data placement and load balanc
 
 For load balancing commands in `yb-admin`, see [Rebalancing commands (yb-admin)](../../../admin/yb-admin/#rebalancing-commands).
 
-For information on internal load balancing to power geo-distributed applications, see [Yugabyte JDBC Driver](../../../integrations/jdbc-driver).
+For information on internal load balancing to power geo-distributed applications, see [YugabyteDB JDBC Driver](/preview/reference/drivers/java/yugabyte-jdbc-reference/.
```
Comment (ddhodge) [inline docs/content/v2.12/reference/configuration/yb-master.md:343]: missing closing )

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/reference/configuration/yb-master.md (hunk 5 above)
Comment (aishwarya24) [reply docs/content/preview/reference/configuration/yb-master.md:321]: I agree , I added this when we introduced the JDBC smart driver. It is not relevant to the information on that page in particular. Will remove it. Thanks.

File: docs/content/preview/integrations/camunda.md (hunk 4 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/camunda.md:175]: @polarweasel added the entry in `netlify.toml`. Thanks!

File: docs/netlify.toml (hunk 1 above)
Comment (polarweasel) [reply docs/netlify.toml:263]: Here's what these two actually need to look like:

```toml
[[redirects]]
  from = "/:version/integrations/*-driver/"
  to = "/preview/reference/drivers/java/yugabyte-jdbc-reference/"
  force = true

[[redirects]]
  from = "/:version/integrations/*"
  to = "/preview/integrations/:splat"
  force = true
```

Comment (aishwarya24) [reply docs/netlify.toml]: Netlify was not recognizing the pattern. Modified it to be more specific and this works as expected. Will merge the PR.

**Pull request merged**